Gruntfile.coffee

```diff
@@ -8,14 +9,15 @@ serverConfigs =
 
 for task, config of serverConfigs
 	config.plugins.push(
-		new webpack.optimize.UglifyJsPlugin(
-			output:
-				beautify: true
-				ascii_only: true
-			compress:
-				sequences: false
-				unused: false # We need this off for OMeta
-			mangle: false
+		new UglifyJsPlugin(
+			uglifyOptions:
+				output:
+					beautify: true
+					ascii_only: true
+				compress:
+					sequences: false
+					unused: false # We need this off for OMeta
```

Comment:
Do we still need this?

Reply:
Yes, we still have ometa -> js compiled code that would be affected